### PR TITLE
Minor updates to english language file

### PR DESCRIPTION
**Pull request by JoyBates**

### CODE REVIEW

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 1]
```diff
@@ -343,7 +343,7 @@
 	"block.securitycraft.redstone_mine": "Redstone Ore Mine",
 	"block.securitycraft.emerald_mine": "Emerald Ore Mine",
 	"block.securitycraft.quartz_mine": "Nether Quartz Ore Mine",
-    "block.securitycraft.reinforced_redstone_lamp": "Reinforced Redstone Lamp",
+    	"block.securitycraft.reinforced_redstone_lamp": "Reinforced Redstone Lamp",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json:346]: Unnecessary change. Please revert the line to how it was before.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 2]
```diff
@@ -498,17 +498,16 @@
 	"gui.securitycraft:scManual.viewActivatedBlock": "View activated: This block can be activated by hovering over it with your cursor while in block-breaking range.",
 	"gui.securitycraft:scManual.explosiveBlock": "Explosive: This block is able to be remotely detonated, and may be able to be activated/deactivated, using the Mine Remote Access Tool.",
 	"gui.securitycraft:scManual.customizableBlock": "Customizable: You may use the Universal Block Modifier and the various modules added by SecurityCraft to customize the functions of this block.",
-    "gui.securitycraft:scManual.options": "Available options:",
-    "gui.securitycraft:scManual.modules": "Available modules:",
+    	"gui.securitycraft:scManual.options": "Available options:",
+    	"gui.securitycraft:scManual.modules": "Available modules:",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json:502]: Unnecessary change. Please revert the line to how it was before.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 3]
```diff
@@ -498,17 +498,16 @@
...
 	"gui.securitycraft:scManual.recipe.keypad": "Place down a Frame and rightclick it with a Key Panel.",
 	"gui.securitycraft:scManual.recipe.keypad_chest": "Place down a Chest and rightclick it with a Key Panel.",
 	"gui.securitycraft:scManual.recipe.keypad_furnace": "Place down a Furnace and rightclick it with a Key Panel.",
 	"gui.securitycraft:scManual.recipe.reinforced": "Any reinforced block can be crafted as follows, if not otherwise specified in this book. Place down the vanilla equivalent (e.g. Reinforced Stone -> Stone) and break it with a Universal Block Reinforcer, or rightclick the Universal Block Reinforcer, put it into the slot and close the GUI.",
 	"gui.securitycraft:scManual.recipe.reinforced_iron_trapdoor": "Any reinforced block can be crafted as follows, if not otherwise specified in this book. Place down the vanilla equivalent (e.g. Reinforced Stone -> Stone) and break it with a Universal Block Reinforcer, or rightclick the Universal Block Reinforcer, put it into the slot and close the GUI.",
-	"gui.securitycraft:scManual.recipe.adminTool": "The Admin Tool is only available via the creative inventory, if it is enabled via the config file.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json:508]: If you don't change [this line](https://github.com/Geforce132/SecurityCraft/blob/1.15.x/src/main/java/net/geforcemods/securitycraft/compat/jei/SCJEIPlugin.java#L22) to use snake_case, then JEI will not show the correct line.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 4]
```diff
@@ -498,17 +498,16 @@
...
 	"gui.securitycraft:scManual.recipe.admin_tool": "The Admin Tool is only available via the creative inventory, if it is enabled via the config file.",
 	"gui.securitycraft:scManual.recipe.reinforced_crystal_quartz": "Made from regular Crystal Quartz Blocks in a Universal Block Reinforcer.",
-	"gui.securitycraft:scManual.recipe.sc_manual": "The SecurityCraft Manual is only available via the creative inventory or by typing \"/sc help\" (without quotes).",
+	"gui.securitycraft:scManual.recipe.sc_manual": "The SecurityCraft Manual is available via the creative inventory, by typing \"/sc help\" (without quotes), or by crafting a book together with iron bars.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json:510]: Unnecessary change because the recipe is shown on the manual page. We don't need an extra part in the text telling the user about that recipe. Please revert the line to how it was before. Apart from that, I don't think this line is used anymore.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 5]
```diff
@@ -585,7 +584,7 @@
 	"gui.securitycraft:blockPocketManager.deactivate": "Deactivate",
 	"gui.securitycraft:blockPocketManager.size": "Size: %sx%sx%s",
 
-	"messages.securitycraft:bugReport": "You can report a crash or a bug using this GoogleDocs form",
+	"messages.securitycraft:bugReport": "You can report a crash or a bug using this GoogleDocs form", // Should there be a link to the form?
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json:587]: No, it's added through code.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 6]
```diff
@@ -625,7 +624,7 @@
 	"messages.securitycraft:srat.cantBind": "You can't bind a sentry that doesn't belong to you.",
 	"messages.securitycraft:srat.bound": "Bound the sentry at # to the remote access tool.",
 	"messages.securitycraft:srat.unbound": "Unbound the sentry at # from the remote access tool.",
-	"messages.securitycraft:universalBlockModifier.cantModify": "\"",
+	"messages.securitycraft:universalBlockModifier.cantModify": "\"", // Only a " ?
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json:627]: It's no longer used.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 7]
```diff
@@ -643,12 +642,12 @@
...
 	"messages.securitycraft:tip.discord": "Join our Discord server to chat and to receive important updates and announcements regarding SecurityCraft!",
-	"messages.securitycraft:tip.scserver": "The official SecurityCraft server is back! Join using this IP: 64.38.249.237",
+	"messages.securitycraft:tip.scserver": "The official SecurityCraft server is back! Join using this IP: securitycraft.nodecraft.gg",
 	"messages.securitycraft:universalKeyChanger.notKeypad": "Rightclick a keypad with the universal key changer to change its passcode.",
 	"messages.securitycraft:universalKeyChanger.passcodeChanged": "The passcode has been changed.",
 	"messages.securitycraft:passwordProtected.notSetUp": "This block doesn't have a password yet. You cannot interact with it.",
 	"messages.securitycraft:sentry.needsBlockBelow": "You have to place the Sentry on top of a block!",
-	"messages.securitycraft:sentry.needsBlocksAround": "Make sure, that your Sentry is surrounded with blocks on all of its four sides!",
+	"messages.securitycraft:sentry.needsBlocksAround": "Make sure that your Sentry is surrounded with blocks on all four sides!", // wasn't this removed?
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json:650]: Yes.

File: src/main/resources/assets/securitycraft/lang/en_us.json (hunk 2 above)
Comment (JoyBates) [reply src/main/resources/assets/securitycraft/lang/en_us.json:502]: Raqbit reeee’d me for not having indentations here, so just checking you’re aware that this is a format change, not a content change.

Comment (JoyBates) [reply src/main/resources/assets/securitycraft/lang/en_us.json:502]: Raqbit reeee’d at me for not having these lines indented, so just verifying with you that you know this is a format/cosmetic change and not a content change. Will still revert if you want, of course.

Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/en_us.json:502]: Yes, I am aware. However you added a tab instead of replacing the 4 spaces with it.  I'll fix indentation myself in these files.
